connector/mqtt/vernemq/broker/bin/LICENSE_NOTICE

```diff
@@ -0,0 +1,10 @@
+##################################################################
```

Comment:
I have some suggestions:

- Change the filename just to NOTICE

- Change the file's content by something like:

To meet the needs of dojot IoT Platform, it is built a customized VerneMQ image. The Dockerfile and scripts used to build this image were based on code from https://github.com/sportebois/docker-vernemq, which are licensed under the Apache 2.0 license.